app/consumers/concerns/report_parsing.rb

```diff
@@ -24,20 +24,30 @@ def enqueue_parse_report_job(reports)
       reports.each do |profile_id, report|
         job = ParseReportJob
               .perform_async(ActiveSupport::Gzip.compress(report), metadata)
-        logger.info("Message enqueued: #{request_id} as #{job}" \
-                    " for #{profile_id}")
-        notify_payload_tracker(:received,
-                               "File is valid. Job #{job} enqueued")
+        notify_report_success(profile_id, job)
       end
 
       validation_payload(request_id, valid: true)
     end
 
     private
 
+    def notify_report_success(profile_id, job)
+      msg = "Enqueued report parsing of #{profile_id}"
+      msg += " from request #{request_id} as job #{job}"
```

Comment:
As we have frozen_string_literal, all strings are immutable and any modification allocates a new one. Using a multiline concatenation like in the original code is slightly ~~harder~~/better/faster.

Reply:
Yes. I've sacrificed the allocation for readability. These could not be frozen as there is an inline formatting, or can they?

Reply:
Well, it definitely allocates more than a single one with a `\` broken to multiple lines...